src/test/scala/uk/gov/nationalarchives/db/users/LambdaSpec.scala

```diff
@@ -0,0 +1,41 @@
+package uk.gov.nationalarchives.db.users
```

Comment:
I can't seem to get the tests to run successfully from the command line, locally following the README instructions. Getting the following error:

`...
[info] LambdaSpec:
[info] The process method
[info] - should create the users with the correct parameters *** FAILED ***
[info]   List() did not equal List("INSERT", "SELECT", "UPDATE") (LambdaSpec.scala:31)
[info]   Analysis:
[info]   Nil$(0: -> INSERT, 1: -> SELECT, 2: -> UPDATE)
[info] Run completed in 764 milliseconds.
[info] Total number of tests run: 1
[info] Suites: completed 1, aborted 0
[info] Tests: succeeded 0, failed 1, canceled 0, ignored 0, pending 0
[info] *** 1 TEST FAILED ***
[error] Failed tests:
[error] 	uk.gov.nationalarchives.db.users.LambdaSpec
[error] (Test / test) sbt.TestsFailedException: Tests unsuccessful
[error] Total time: 1 s, completed 17 Mar 2021, 08:19:34
...`

Reply:
Ah yeah, same. It's because I'd changed the name of the user in the main Lambda but not in the tests.